### PR TITLE
Update Move-CsUser.md

### PR DESCRIPTION
**Pull request by bjwhalen**
updated since MoveToTeams switch no longer required.

### CODE REVIEW

File: skype/skype-ps/skype/Move-CsUser.md [hunk 1]
```diff
@@ -66,19 +56,25 @@ When moving a user to or from Office 365 (either Skype for Business Online or Te
...
 ```
 
-In Example 2, the Move-CsUser cmdlet is used to move the user account with sip address PilarA@contoso.com to Skype for Business Online. This is the same cmdlet usage as example 1, except the MoveToTeams switch is not specified.
+And then from a TeamsPowerShell window, remove TeamsOnly mode by running:
+
+```powershell
+Grant-CsTeamsUpgradePolicy -Identity PilarA@contoso.com -PolicyName $null
+```
+
+In Example 2, the Move-CsUser cmdlet is first used to move the user account with sip address PilarA@contoso.com to TeamsOnly in Microsoft 365. Then, using Teams PowerShell  the mode is updated to remove TeamsOnly mode so the user can use Skype for Business Online.
```
Comment (dariomws) [inline skype/skype-ps/skype/Move-CsUser.md]:
```suggestion
In Example 2, the Move-CsUser cmdlet is first used to move the user account with the sip address PilarA@contoso.com to TeamsOnly in Microsoft 365. Then, using Teams PowerShell, the mode is updated to remove TeamsOnly mode so the user can use Skype for Business Online. 
```

File: skype/skype-ps/skype/Move-CsUser.md [hunk 2]
```diff
@@ -151,7 +147,7 @@ Accept wildcard characters: False
 
 ### -MoveToTeams
 
-If specified, the user will be moved to Office 365 as a Teams-only user. This will ensure incoming chats and calls land in the user's Teams client. This parameter is only available with Skype for Business Server 2019 and CU8 for Skype for Business Server 2015.
+This parameter is no longer needed. This parameter is only  available with Skype for Business Server 2019 and CU8 for Skype for Business Server 2015 and previously was required to move a user *directly* to TeamsOnly in Microsoft 365. However, when using Move-CsUser, users are now always moved to TeamsOnly, whether this switch is specified or not.
```
Comment (dariomws) [inline skype/skype-ps/skype/Move-CsUser.md]:
```suggestion
This parameter is no longer needed. This parameter is only available with Skype for Business Server 2019 and CU8 for Skype for Business Server 2015 and previously was required to move a user *directly* to TeamsOnly in Microsoft 365. However, when using Move-CsUser, users are now always moved to TeamsOnly, whether this switch is specified or not.
```

**Review by dariomws — CHANGES_REQUESTED** [review]

File: skype/skype-ps/skype/Move-CsUser.md [hunk 3]
```diff
@@ -184,7 +182,7 @@ Accept wildcard characters: False
 
 ### -HostedMigrationOverrideUrl
 
-The hosted migration service is the service in Office 365 that performs user moves. By default, there is no need to specify a value for this parameter, as long as the hosting provider has its AutoDiscover URL properly configured and you are using an admin account the ends in .onmicrosoft.com. If you are using a user account from on-premises that synchronized to the cloud, you must specify this parameter.  See [Required administrative credentials](https://docs.microsoft.com/skypeforbusiness/hybrid/move-users-between-on-premises-and-cloud#required-administrative-credentials).
+The hosted migration service is the service in Office 365 that performs user moves. By default, there is no need to specify a value for this parameter, as long as the hosting provider has its AutoDiscover URL properly configured and you are using an admin account the ends in .onmicrosoft.com. If you are using a user account from on-premises that synchronized to the cloud, you must specify this parameter. See [Required administrative credentials](https://docs.microsoft.com/skypeforbusiness/hybrid/move-users-between-on-premises-and-cloud#required-administrative-credentials).
```
Comment (dariomws) [inline skype/skype-ps/skype/Move-CsUser.md:185]:
```suggestion
The hosted migration service is the service in Office 365 that performs user moves. By default, there is no need to specify a value for this parameter, as long as the hosting provider has its AutoDiscover URL properly configured and you are using an admin account the ends in .onmicrosoft.com. If you are using a user account from on-premises that synchronized to the cloud, you must specify this parameter. See [Required administrative credentials](/skypeforbusiness/hybrid/move-users-between-on-premises-and-cloud#required-administrative-credentials).
```

**Review by dariomws — CHANGES_REQUESTED** [review]

**Review by dariomws — APPROVED** [review]
Thank you @bjwhalen and @chrisda!

**Review by get-itips — COMMENTED** [review]
Reviewed

**Pull request merged**